docs/character-ref-attributes.adoc

```diff
@@ -1,7 +1,12 @@
 [[char-ref-table]]
-= Built-in attributes for character replacements
+//= Built-in attributes for character replacements
+= Predefined attributes
```

Comment:
I kind of like the term "Built-in" rather than "Predefined". I think "predefined" are what we should call the character replacement attributes. In other words:

* built-in - reserved attribute names that affect processing in some way
* predefined - attributes that have been preset for convenience, but are otherwise just data templates

wdyt?